src/store/actions/list.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ALL_LIST_SAGA,
+  ADD_LIST_SAGA,
+  EDIT_LIST_SAGA,
+  DELETE_LIST_SAGA,
+} from "../types";
+
+export const allList = () => {
```

Comment:
In es6 you could write this method like this

export const allList = () => ({
    type: ALL_LIST_SAGA
})

Reply:
excluding the return statement making it less to code

Reply:
done